app/routes/_layout+/assets.$assetId.overview.assign-custody.tsx

```diff
@@ -44,11 +46,11 @@ export async function loader({ context, request, params }: LoaderFunctionArgs) {
   });
 
   try {
-    const { organizationId } = await requirePermission({
+    const { organizationId, role } = await requirePermission({
       userId,
       request,
       entity: PermissionEntity.asset,
-      action: PermissionAction.update,
+      action: [PermissionAction.update, PermissionAction.custody],
```

Comment:
This will work but you dont need `update` here.

ADMIN & OWNER can do everything.
Self service will have custody
Base will not be able to do anything as they dont have custody